code/datums/repositories/decls.dm

```diff
@@ -14,8 +14,7 @@
 //     III. Thou shalt not write a decl that relies on arguments supplied to New().
 //     IV.  Thou shalt not call Initialize() on a /decl.
 
-var/repository/decls/decls_repository // Initialiozed in /datum/global_init/New()
-
+var/repository/decls/decls_repository = new()
```

Comment:
What is the purpose of moving it to world new?

Reply:
Lazy fix for it needing to exist in global_init because I didn't bother removing it from GLOB or checking to update literally the only usage of the list